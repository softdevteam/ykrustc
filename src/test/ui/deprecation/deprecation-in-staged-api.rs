#![feature(staged_api)]
#![stable(feature = "stable_test_feature", since = "1.0.0")]
#[deprecated] //~ ERROR `#[deprecated]` cannot be used in staged API
fn main() {}
