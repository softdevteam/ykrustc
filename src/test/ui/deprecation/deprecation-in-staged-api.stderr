error: `#[deprecated]` cannot be used in staged API
  --> $DIR/deprecation-in-staged-api.rs:3:1
   |
LL | #[deprecated]
   | ^^^^^^^^^^^^^ use `#[rustc_deprecated]` instead
LL | fn main() {}
   | ------------

error: aborting due to previous error

